### PR TITLE
Usermodels

### PR DESCRIPTION
**Pull request by jackbackes**

### CODE REVIEW

File: server/db/models/address.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var _ = require('lodash');
+var Sequelize = require('sequelize');
+
+module.exports = function (db) {
+
+    db.define('address', {
```
Comment (jonperrelle) [inline server/db/models/address.js:8]: Should the address categories all be required?

File: server/db/models/review.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    db.define('review', {
+      rating: {
+        type: Sequelize.INTEGER,
+        validate: {
+          min: 0,
+          max: 5,
+          notNull: true
+        }
+      },
+      description: {
```
Comment (jonperrelle) [inline server/db/models/review.js:17]: Reviews need to be at least a certain amount of characters (per the workshop).  Add a validate with a len validate of [40]?

File: server/db/models/order.js [hunk 3]
```diff
@@ -6,6 +6,21 @@ var Sequelize = require('sequelize');
...
+      billingHistory: {
+        type: Sequelize.ARRAY(Sequelize.JSONB)
+      },
+      transactionSuccessful: {
+        type: Sequelize.BOOLEAN
+      },
+      total: {
+        type: Sequelize.DECIMAL
+      }
     })
```
Comment (jonperrelle) [inline server/db/models/order.js:25]: Not sure what billing history does?

File: server/db/models/address.js (hunk 1 above)
Comment (geoffbass) [inline server/db/models/address.js:8]: fixed

File: server/db/models/review.js (hunk 2 above)
Comment (geoffbass) [inline server/db/models/review.js:17]: made it [5] for now, open to changing it

**Pull request closed without merging**